roles/restic/defaults/main.yml

```diff
@@ -0,0 +1,64 @@
+---
+
+restic_version: "0.16.0"
+restic_job_description: "Run restic backup job"
+restic_binary: "/usr/local/bin/restic"
+
+restic_repository: ~
+restic_password: ~
+
+restic_backup_paths: []
+restic_backup_commands: []
+
+restic_systemd_user: root
+restic_systemd_unit_name: "restic"
+restic_systemd_service_name: "{{ restic_systemd_unit_name }}.service"
+restic_systemd_timer_name: "{{ restic_systemd_unit_name }}.timer"
+restic_systemd_unit_file_directory: "/etc/systemd/system"
+restic_systemd_working_directory: "/tmp"
+restic_systemd_syslog_identifier: "{{ restic_systemd_unit_name }}"
+
+restic_systemd_timer_on_calendar: daily
+restic_systemd_timer_accuracy_sec: 3600
+restic_systemd_timer_persistent: true
+restic_systemd_timer_description: >-
+  Run {{ restic_systemd_unit_name }}.service on schedule
+
+restic_default_environment:
+  RESTIC_REPOSITORY: "{{ restic_repository }}"
+  RESTIC_PASSWORD: "{{ restic_password }}"
+restic_environment: {}
+restic_systemd_environment: >-2
+  {{ restic_default_environment | combine(restic_environment) }}
+
+restic_system_type: "{{ ansible_facts['system'] | lower }}"
+restic_system_arch: >-
+  {{ restic_system_arch_map_ansible_arch[ansible_facts['architecture']] | default('amd64') }}
+restic_system_arch_map_ansible_arch:
+  x86_64: amd64
+  aarch64: arm64
+  arm64: arm64
+  armv6l: arm
+  armv6: arm
+  armv7l: arm
+  armv7: arm
+  armv8l: arm64
+  armv8: arm64
+  i386: 386
```

Comment:
This being a number and not a string is a bit icky, but shouldn't cause any problems

Reply:
I know. in the key, it wouldve caused problems - i don't really _like_ this but for now i will merge it.